### ISSUE TITLE
Allow passing milestone to issue

### ISSUE THREAD

**Issue by bengry** [JasonEtco/create-an-issue#43]
I want to automatically create an issue once a milestone is created, I have the following workflow:
```yml
...
- name: create release-driving issue
  uses: JasonEtco/create-an-issue@v2.2.0
  with:
    filename: .github/issue_templates/RELEASE_ISSUE_TEMPLATE.md
    assignees: ${{ github.actor }}
    milestone: ${{ github.event.milestone.number }}
  env:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

with `RELEASE_ISSUE_TEMPLATE.md` having the following content:
```md
---
title: ...
labels: release
milestone: {{ tools.inputs.milestone }}
---
```

This doesn't work, and I get the following error when running the workflow:
> ✖  error     An error occurred while creating the issue. This might be caused by a malformed issue title, or a typo in the labels or assignees. Check .github/issue_templates/RELEASE_ISSUE_TEMPLATE.md!
✖  error     HttpError: Invalid value for parameter 'milestone': {"[object Object]":null} is NaN 

I guess this is because the template is not being passed the milestone input, even though it seems like it should work, following https://github.com/JasonEtco/actions-toolkit#toolsinputs

**Comment by JasonEtco** [comment]
Hey @bengry 👋 thanks for the issue. I agree that it _should_ work like this, but unfortunately it does not. We have to grab the inputs specifically, there's no "catch-all":

https://github.com/JasonEtco/create-an-issue/blob/696e3cb79a9719ced6cbe8decfdc31cd74639096/index.js#L13-L14

`tools.inputs` wouldn't be able to return an object of them all either because of how the inputs are exposed in the runtime (`process.env.EXAMPLE`).

However, I'd be totally open to a PR that does this:

```js
const milestone = core.getInput('milestone') || attributes.milestone
```

**Comment by bengry** [comment]
Great, thanks. I'll send a PR when I get to the office. Also, why is the `|| attributes.milestone` needed at all though?

**Comment by JasonEtco** [comment]
Good question! `attributes` comes from the front-matter of the markdown file. You could also define a `milestone` there:

```markdown
---
title: ...
labels: release
milestone: 123
---

We don't pass `tools` to the template itself like what you've done above; instead, we grab the `attributes` (so the parsed front-matter) and include those in the API request to create the issue:

https://github.com/JasonEtco/create-an-issue/blob/696e3cb79a9719ced6cbe8decfdc31cd74639096/index.js#L18-L21

We can't directly pass `tools.inputs` either, because `inputs`has to be an object of strings, where sometimes we want arrays.

**Comment by JasonEtco** [comment]
Closed by #44
